src/data/industrial-2020.json

```
@@ -0,0 +1,460 @@
+  },
+  {
+    "id": "91.14",
+    "materia": "Informática para la Gestión de Empresas",
+    "creditos": 4,
+    "correlativas": "ID19",
+    "categoria": "Materias Electivas"
+  },
+  {
+    "id": "91.27",
+    "materia": "Logística y Supply Chain",
+    "creditos": 4,
+    "correlativas": "ID15",
+    "categoria": "Materias Electivas"
+  },
+  {
+    "id": "91.51",
+    "materia": "Desarrollo y Gestión de Proyectos e Innovación",
+    "creditos": 4,
+    "correlativas": "ID15",
+    "categoria": "Materias Electivas"
+  },
+  {
+    "id": "91.37",
+    "materia": "Emprendimientos en Ingeniería",
+    "creditos": 4,
+    "requiere": 138,
+    "requiereCBC": true,
+    "categoria": "Materias Electivas"
+  },
+  {
+    "id": "92.08",
+    "materia": "Diseño de Producto",
+    "creditos": 4,
+    "correlativas": "ID15-92.01",
+    "categoria": "Materias Electivas"
+  },
+  {
+    "id": "ID42",
+    "materia": "Diseño de Producto Digital",
+    "creditos": 4,
+    "correlativas": "ID19",
+    "categoria": "Materias Electivas"
+  },
+  {
+    "id": "92.24",
+    "materia": "Ingeniería y Desarrollo de Envases y Embalajes",
+    "creditos": 4,
+    "requiere": 138,
+    "requiereCBC": true,
+    "categoria": "Materias Electivas"
+  },
+  {
+    "id": "ID44",
+    "materia": "Agroindustria y Bioeconomía",
+    "creditos": 4,
+    "requiere": 138,
+    "requiereCBC": true,
+    "categoria": "Materias Electivas"
+  },
+  {
+    "id": "92.09",
+    "materia": "Biopolímeros e Industrias Plásticas",
+    "creditos": 4,
+    "correlativas": "92.18",
+    "categoria": "Materias Electivas"
+  },
+  {
+    "id": "92.20",
+    "materia": "Industria Automotriz",
+    "creditos": 4,
+    "correlativas": "ID15",
+    "categoria": "Materias Electivas"
+  },
+  {
+    "id": "92.10",
+    "materia": "Industrias de Celulosa y Papel",
+    "creditos": 4,
+    "correlativas": "92.18",
+    "categoria": "Materias Electivas"
+  },
+  {
+    "id": "92.15",
+    "materia": "Industrias de la Alimentación",
+    "creditos": 4,
+    "correlativas": "92.18",
+    "categoria": "Materias Electivas"
+  },
+  {
+    "id": "92.12",
+    "materia": "Industrias Textiles",
+    "creditos": 4,
+    "correlativas": "ID15-92.01",
+    "categoria": "Materias Electivas"
+  },
+  {
+    "id": "ID50",
+    "materia": "Mantenimiento e Ingeniería de Planta",
+    "creditos": 4,
+    "correlativas": "ID15",
+    "categoria": "Materias Electivas"
+  },
+  {
+    "id": "92.13",
+    "materia": "Materiales y Aplicaciones II",
+    "creditos": 4,
+    "correlativas": "92.01",
+    "categoria": "Materias Electivas"
+  },
+  {
+    "id": "92.00",
+    "materia": "Tesis de Ingeniería Industrial",
+    "creditos": 13,
+    "requiere": 178,
+    "requiereCBC": true,
+    "categoria": "Fin de Carrera"
+  },
+  {
+    "id": "92.99",
+    "materia": "Trabajo Profesional de Ingeniería Industrial",
+    "creditos": 13,
+    "requiere": 178,
+    "requiereCBC": true,
+    "categoria": "Fin de Carrera"
+  }
```

```suggestion
  {
    "id": "92.XX",
    "materia": "Tesis o Trabajo Profesional de Ingeniería Industrial",
    "creditos": 12,
    "requiere": 140,
    "requiereCBC": true,
    "categoria": "Fin de Carrera (Obligatorio)"
  }
```

Por ahora lo dejaria todo junto para mas claridad